org.lflang/src/org/lflang/federated/extensions/CppExtension.java

```diff
@@ -0,0 +1,66 @@
+package org.lflang.federated.extensions;
```

Comment:
Its currently not planned to support federates (using this scheme) in C++. I suggest removing this stub